.github/workflows/ci.yml

```diff
@@ -14,6 +14,7 @@ on:
     types: [opened, synchronize]
     paths-ignore:
       - "**/*.md"
+      - "website/**/*"
```

Comment:
If we use `paths-ignore` here, the required CI will never be passed:

![image](https://github.com/web-infra-dev/rspack/assets/7237365/e8fc15de-f44e-4363-87bb-1d8b53d703c7)

Reply:
that's why Rsbuild uses an extra script to support this

https://github.com/web-infra-dev/rsbuild/blob/main/scripts/skipCI.js